### ISSUE TITLE
Refactoring Bootstrap library

### ISSUE THREAD

**Issue by jghjianghan** [jghjianghan/South-Korea-Covid19-IS#19]
Saat ini halaman pengunjung menggunakan bootstrap 4, sedangkan halaman admin menggunakan bootstrap 5.
Ketika di-merge, halaman pengunjung menjadi broken jika link bootstrap 4 dihapus. Sepertinya kl bootstrap 5 dihapus halaman admin akan menjadi broken (@obedkristiaji @mariqpratama tolong konfirmasi di comment). Saat ini 2-2nya masih diimport di view/layout/layout.php

Solusi:
- keep salah satu, class menyesuaikan
- Keep 2-2nya, tapi import berdasarkan jenis halamannya, caranya kurang lebih seperti #14 .

**Comment by obedkristiaji** [comment]
iya, ketika bootstrap 5 dihapus halaman admin menjadi broken, jadi tadi diakali dengan tetap pakai 2-2nya di si layout terus kita sedikit mengatur lagi si halaman adminnya agar ketika memakai 2-2nya jadi tidak broken

**Comment by jghjianghan** [comment]
Ternyata ada masalah baru. Menurut keterangan dari @johan-wiguna :
- Kalau Bootstrap 4 dihapus dari halaman admin, dropdown-nya tidak berfungsi.
- Kalau Bootstrap 5 dihapus dari halaman pengunjung, card tidak akan berfungsi (karena card dihapus pada Bootstrap 5).

Opsi saat ini:
- Biarkan saja mengimport 2 Bootstrap
- Memilih salah satu versi Bootstrap dan menyesuaikan class-class pada salah satu halaman.
